v7/flag_overrides.go

```diff
@@ -0,0 +1,136 @@
+package configcat
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/configcat/go-sdk/v7/internal/wireconfig"
+	"io"
+	"io/ioutil"
+)
+
+// OverrideBehaviour describes how the overrides should behave.
+type OverrideBehaviour int
+
+const (
+	// LocalOnly means that when evaluating values, the SDK will not use feature flags and settings from the
+	// ConfigCat CDN, but it will use all feature flags and settings that are loaded from local-override sources.
+	LocalOnly = 0
+
+	// LocalOverRemote means that when evaluating values, the SDK will use all feature flags and settings that are
+	// downloaded from the ConfigCat CDN, plus all feature flags and settings that are loaded from
+	// local-override sources. If a feature flag or a setting is defined both in the fetched and the local-override
+	// source then the local-override version will take precedence.
+	LocalOverRemote = 1
+
+	// RemoteOverLocal means when evaluating values, the SDK will use all feature flags and settings that are
+	// downloaded from the ConfigCat CDN, plus all feature flags and settings that are loaded from local-override
+	// sources. If a feature flag or a setting is defined both in the fetched and the local-override source then the
+	// fetched version will take precedence.
+	RemoteOverLocal = 2
+)
+
+// FlagOverrides describes feature flag and setting overrides. With flag overrides you can overwrite the
+// feature flags downloaded from the ConfigCat CDN with local values.
+//
+// With Values, you can set up the SDK to load your feature flag overrides from a map.
+//
+// With FilePath, you can set up the SDK to load your feature flag overrides from a JSON file.
+type FlagOverrides struct {
+	// Behaviour describes how the overrides should behave. Default is LocalOnly.
+	Behaviour OverrideBehaviour
+
+	// Values is a map that contains the overrides.
+	Values map[string]interface{}
+
+	// FilePath is the path to a JSON file that contains the overrides.
```

Comment:
This should document the expected schema of the JSON.

Reply:
I'll put a link here that'll point to the online docs which'll contain the schema documentation